src/views/settings/Terminal/Endpoint/EndpointCreateUpdate.vue

```diff
@@ -23,7 +23,7 @@ export default {
         [
           this.$t('Port'),
           [
-            'http_port', 'https_port', 'ssh_port', 'rdp_port'
+            'http_port', 'https_port', 'ssh_port', 'rdp_port','vnc_port'
           ]
         ],
         [
```

Comment:
The code does not have any known issues based on the information provided, but it could be simplified further to include all protocols with one key-value pair instead of four.

```yaml
  - "PORTS":
        -
          "HTTP PORT": "<http_port>"
```